src/app/repositories/user.repository.ts

```diff
@@ -3,4 +3,4 @@ import { AppDataSource } from "@root/ormconfig";
 
 const userRepository = AppDataSource.getRepository(User);
 
-export { userRepository };
+export { User, userRepository };
```

Comment:
[Q] Why export the `User` entity?

Reply:
[Request for change] Remove `User` entity export
```suggestion
export { userRepository };
```